platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -801,7 +801,10 @@ public class SwirldsPlatform implements Platform {
             loadStateIntoConsensus(initialState);
 
             platformWiring.updateNonAncientEventWindow(NonAncientEventWindow.createUsingPlatformContext(
-                    initialState.getRound(), initialMinimumGenerationNonAncient, platformContext));
+                    initialState.getRound(),
+                    initialMinimumGenerationNonAncient,
+                    initialMinimumGenerationNonAncient,
```

Comment:
Could you explain why the min gen non expired here is the same as `initialMinimumGenerationNonAncient`?

Reply:
There's actually a good reason for this. Added the following comment:

```
// We only load non-ancient events during start up, so the initial non-expired event window will be
// equal to the non-ancient event window when the system first starts. Over time as we get more events,
// the non-expired event window will continue to expand until it reaches its full size.
```